ivy/functional/ivy/statistical.py

```diff
@@ -413,6 +405,82 @@ def mean(
     return current_backend(x).mean(x, axis=axis, keepdims=keepdims, out=out)
 
 
+@to_native_arrays_and_back
+@handle_out_argument
+@handle_nestable
+@handle_exceptions
+def min(
+    x: Union[ivy.Array, ivy.NativeArray],
+    /,
+    *,
+    axis: Union[int, Tuple[int]] = None,
+    keepdims: bool = False,
+    out: Optional[ivy.Array] = None,
+) -> ivy.Array:
+    """Calculates the minimum value of the input array x.
+
+    .. note::
+    When the number of elements over which to compute the minimum value is zero, the
+    minimum value is implementation-defined. Specification-compliant libraries may
+    choose to raise an error, return a sentinel value (e.g., if x is a floating-point
+    input array, return NaN), or return the maximum possible value for the input array x
+    data type (e.g., if x is a floating-point array, return +infinity).
+
+    **Special Cases**
+
+    For floating-point operands,
+
+    If x_i is NaN, the minimum value is NaN (i.e., NaN values propagate).
+
+    Parameters
+    ----------
+    x
+        Input array containing elements to min.
+    axis
+         axis or axes along which minimum values must be computed. By default, the
+         minimum value must be computed over the entire array. If a tuple of integers,
+         minimum values must be computed over multiple axes. Default: None.
+    keepdims
+        optional boolean, if True, the reduced axes (dimensions) must be included in the
+        result as singleton dimensions, and, accordingly, the result must be compatible
+        with the input array (see Broadcasting). Otherwise, if False, the reduced axes
+        (dimensions) must not be included in the result. Default: False.
+    out
+        optional output array, for writing the result to.
+
+    Returns
+    -------
+    ret
+        if the minimum value was computed over the entire array, a zero-dimensional
+        array containing the minimum value; otherwise, a non-zero-dimensional array
+        containing the minimum values. The returned array must have the same data type
+        as x.
+
+    Examples
+    --------
+
+    With :code:`ivy.Container` input:
```

Comment:
Please read the documentation about the type of example expected here as this is incorrect.

Reply:
Yeah, for some reason (might be my mistake, not sure though) there were 2 min functions in statistical, and that was the reason why it failed the linter check. Installed and run the "pre-commit" in `ivy/functional/ivy/statistical.py`. Eventually, got green on linter check. Then modified the examples. So forth, requesting a new re-review please.